app/views/events/_fields.slim

```diff
@@ -1,7 +1,21 @@
-  = f.text_field :name
-  = f.text_field :location
-  = f.datetime_select :start_date
-  = f.text_field :description
-  = f.text_field :hashtag
-  = f.text_field :interest_list, value: @event.interest_list.join(',')
-  = f.submit
+.Form
+  .Form-label
+    = f.label :name_name
+  = f.text_field :name, class: 'Form-input', required: true
+  .Form-label
+    = f.label :event_hashtag
+  = f.text_field :hashtag, class: 'Form-input Form--hashtag'
```

Comment:
Modifiers are rarely used. When we are using a modifier, in some cases, can this be a smell to refactor the components in small components?